app/assets/stylesheets/views/_loginView.scss

```diff
@@ -0,0 +1,20 @@
+@import 'utilities/variables';
```

Comment:
this is unnecessary

Reply:
if we no import the variables, rails no find them, and send this in the browser, anyway that happen if we put the import in the main.scss

![image](https://user-images.githubusercontent.com/50371065/214726332-ac0a0a73-c7a2-4174-924b-38bb23487881.png)
![image](https://user-images.githubusercontent.com/50371065/214726332-ac0a0a73-c7a2-4174-924b-38bb23487881.png)

Reply:
how looks your principal file of scss?

Reply:
reolved